requirements.txt

```diff
@@ -1,5 +1,6 @@
 geojson>=1.3.2
 python-dateutil>=2.6.0
+natsort==5.1.0
```

Comment:
Can this be a more abstract version? I'd like to move away from using locked down requirements for packages (except in a lockfile, which we don't actually have).